client/keys/utils.go

```diff
@@ -71,10 +59,17 @@ func printInfos(w io.Writer, infos []cryptokeyring.Info, output string) {
 	}
 }
 
-func printTextInfos(w io.Writer, kos []cryptokeyring.KeyOutput) {
-	out, err := yaml.Marshal(&kos)
-	if err != nil {
-		panic(err)
+func printTextInfos(w io.Writer, kos []*cryptokeyring.KeyOutput) {
+	for _, ko := range kos {
+		if ko == nil {
+			continue
+		}
+
+		out, err := yaml.Marshal(ko)
```

Comment:
If introduces a linter issue if you use `yaml.Marshal(&ko)` inside for loop. That's why I decided loop over slice of KeyOutput pointers.